### PR TITLE
support openfunction framework

### PR DESCRIPTION
**Pull request by wanjunlei**

### CODE REVIEW

File: functions-framework-invoker/src/main/java/dev/openfunction/invoker/context/UserContext.java [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class UserContext implements Context {
+
+    private static final Logger logger = Logger.getLogger("dev.openfunction.invoker");
+
+    private static final Map<String, Boolean> bindingQueueComponents = Map.of(
```
Comment (benjaminhuo) [inline functions-framework-invoker/src/main/java/dev/openfunction/invoker/context/UserContext.java:32]: this hardcode bindingqueuecomponents is only used to determine if it's tracable?
@tpiperatgod Go FF has similar logic or use something else?

File: functions-framework-invoker/src/main/java/dev/openfunction/invoker/context/UserContext.java [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+import dev.openfunction.functions.*;
+import io.cloudevents.CloudEvent;
+import io.dapr.client.DaprClient;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class UserContext implements Context {
```
Comment (benjaminhuo) [inline functions-framework-invoker/src/main/java/dev/openfunction/invoker/context/UserContext.java:28]: I think the `UserContext` can just be renamed to `Context` including class name and filename and variable name(lowercase)

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

File: samples/src/main/resources/function/http-with-output.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+apiVersion: core.openfunction.io/v1beta1
+kind: Function
+metadata:
```
[the remark this answers is not in the document]
Comment (wanjunlei) [reply samples/src/main/resources/function/http-with-output.yaml:3]: The samples and docs will be completed later.

File: functions-framework-invoker/src/main/java/dev/openfunction/invoker/context/UserContext.java (hunk 2 above)
Comment (benjaminhuo) [reply functions-framework-invoker/src/main/java/dev/openfunction/invoker/context/UserContext.java:28]: Then we can use FuncContext instead?

Comment (wanjunlei) [reply functions-framework-invoker/src/main/java/dev/openfunction/invoker/context/UserContext.java:28]: There has a class named FunctionContex.

Comment (benjaminhuo) [reply functions-framework-invoker/src/main/java/dev/openfunction/invoker/context/UserContext.java:28]: what's the difference between FunctionContex and UserContext?

Comment (wanjunlei) [reply functions-framework-invoker/src/main/java/dev/openfunction/invoker/context/UserContext.java:28]: The `FunctionContex` is the structure of `FUNC_CONTEXT`, `UserContext` implement the `Context` interface and will pass to the function.

Comment (benjaminhuo) [reply functions-framework-invoker/src/main/java/dev/openfunction/invoker/context/UserContext.java:28]: That's ok if user function can only see context interface

**Pull request merged**